src/plugins/kernel_install/utils.sh

```diff
@@ -550,32 +550,85 @@ function do_uninstall()
   fi
 }
 
+# Element existence check
+# Checks if element belongs to given array
+#
+# @target_element element being checked
+# @array target array
```

Comment:
Use colon : between the argument name and the description, as in `# @target_element: element being checked`.